packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -512,3 +534,32 @@ class _RenderCupertinoPickerSemantics extends RenderProxyBox {
     controller.removeListener(_handleScrollUpdate);
   }
 }
+
+class _CupertinoPickerListWheelChildDelegateWrapper implements ListWheelChildDelegate {
+  _CupertinoPickerListWheelChildDelegateWrapper(
+    this._wrapped, {
+    required this.onTappedChild,
+  });
+  final ListWheelChildDelegate _wrapped;
+  final void Function(int index)? onTappedChild;
+
+  @override
+  Widget? build(BuildContext context, int index) {
+    final Widget? child = _wrapped.build(context, index);
+    if (child == null || onTappedChild == null) {
+      return child;
+    }
+    return GestureDetector(
+      behavior: HitTestBehavior.translucent,
+      excludeFromSemantics: true,
+      onTap: () => onTappedChild!(index),
+      child: child,
+    );
+  }
+  @override
```

Comment:
Turbo nit, this is my own opinion feel free to ignore:
I think it looks better if there's an empty line above a `@override`.